multinode/bootstrapping.md

```diff
@@ -0,0 +1,190 @@
+setting up the extension.
+
+### Validating the database
+
+The database will be validated by checking that the encoding,
+character type, and collation are the same as the current database on
+the access node.
+
+## Setting up the extension
+
+The `timescaledb` extension needs to be installed in the database on
+the data node, so the next step is ensuring that there is a valid
+extension installed in the database by either:
+
+* Creating the extension.
+* Verifying that the extension is compatible with the extension
+  installed on the access node.
+
+### Creating the schema for the extension
+
+To install the extension, a schema is created to store the objects
+that the `timescaledb` extension needs. The schema name is decided by
+looking at what schema name is used for the extension on the access node and
+using the same name on the data node. If you have created the
+`timescaledb` extension in the default manner, `public` will be used
+for the schema on the access node, hence `public` will be used on the
+data node as well.
+
+If the schema name for the extension on the access node is `public`,
+no new schema is created, but otherwise it is expected that the schema
+name does *not* exist. In the event that the schema name exists (but
+is not `public`), an error will be raised and the bootstrapping will abort.
+
+To change the schema used on the access node and data node, you can
+create a new schema, then create the extension with the schema that
+you want:
+
+```
+CREATE SCHEMA magic;
+CREATE EXTENSION timescaledb SCHEMA magic;
+```
+
+When validating the schema, it is necessary that the schema is owned
+by the current user, or is `public`.
+
+>:TIP: Creating the extension in a non-default schema will
+> automatically place all TimescaleDB functions into that
+> schema. Therefore, all function references must be qualified within
+> that schema. For example, to call `create_hypertable` from the
+> previous magic schema:
+> 
+> ```
+> SELECT * FROM magic.create_hypertable('conditions', 'time', 3);
+> ```
+
+### Installing the extension
+
+After the schema is created, the extension is installed using the
+`CREATE EXTENSION` command with the `SCHEMA` option to refer to the
+schema from the section above. The schema name for the extension will
+be either `public` or a dedicated schema name.
+
+If the extension already exists, it is validated to ensure that it can
+be used with the access node and the procedure then sets the metadata.
+
+### Validating the extension
+
+Validation of the extension checks that the extension on the data node
+is of a version that is compatible with the version on the access node
+and that the access node has permissions to use the extension.
+
+The extension on the data node is compatible with the extension on the
+access node if the major versions are the same, and the minor version
+on the data node is before or the same as the minor version on the
+access node. For example:
```

Comment:
```suggestion
access node. The difference in the patch version doesn't affect compatibility. For example:
```
The example demonstrates that patch version can be any, so it is good to mention.